packages/python-google-compute-engine/google_compute_engine/networking/network_setup/tests/network_setup_test.py

```diff
@@ -67,6 +67,72 @@ def testEnableIpv6(self, mock_call):
     ]
     self.assertEqual(mocks.mock_calls, expected_calls)
 
+  @mock.patch('google_compute_engine.networking.network_setup.network_setup.subprocess.check_call')
+  def testDisableIpv6(self, mock_call):
+    mocks = mock.Mock()
+    mocks.attach_mock(mock_call, 'call')
+    mocks.attach_mock(self.mock_logger, 'logger')
+    mocks.attach_mock(self.mock_distro_utils.EnableIpv6, 'enable')
+    mocks.attach_mock(self.mock_distro_utils.DisableIpv6, 'disable')
+    expected_calls = []
+
+    # Clean run, run disable once e.g. at boot.
+    network_setup.NetworkSetup.DisableIpv6(
+        self.setup, ['A'])
+    self.assertEqual(self.setup.ipv6_interfaces, set([]))
+    # No more disables allowed, have to follow the contract of Enable and then
+    # Disable.
+    network_setup.NetworkSetup.DisableIpv6(
+        self.setup, ['A'])
+    expected_calls.extend([mock.call.logger.info(mock.ANY, ['A']),
```

Comment:
Please line wrap after the first open paren (apply this everywhere in the file).